examples/webgl2_volume_cloud.html

```diff
@@ -213,20 +237,31 @@
 						threshold: { value: 0.25 },
 						opacity: { value: 0.25 },
 						range: { value: 0.1 },
-						steps: { value: 100 }
+						steps: { value: 100 },
+						frame: { value: 0 }
+					},
+					defines: {
+						JITTER: true
 					},
 					vertexShader,
 					fragmentShader,
 					side: THREE.BackSide,
 					transparent: true
 				} );
+				material.glslVersion = '300 es';
```

Comment:
Please use `THREE.GLSL3` in this line.

Using this constant instead of adding the version qualifier in the shader code should also be done in `webgl2_volume_perlin`.

Reply:
I'll clean it up 👌

Reply:
https://github.com/mrdoob/three.js/commit/b44e670887c84bbd85296e108024c22682702dd0

Reply:
Sorry, I was in holidays. Thanks for the cleanup!